README.md

```diff
@@ -16,7 +16,48 @@ Further documentation will be available on
 # Please note
 This version works only with `python 3.6.x`
```

Comment:
I would move this below, it is not that relevant to have it at the top

Reply:
I'll fix it